docs/snippets/ov_preprocessing.py

```diff
@@ -214,17 +214,19 @@ def custom_abs(output: Output):
 
 # ======== Step 3: Save the model ================
 # First method - using serialize runtime wrapper
-serialize(model, '/path/to/some_model_saved.xml', '/path/to/some_model_saved.bin')
+serialize(model, xml_path)
 
 # Second method - using Manager and Serialize pass
 manager = Manager()
-manager.register_pass(Serialize('/path/to/some_model_saved.xml', '/path/to/some_model_saved.bin'))
+manager.register_pass(Serialize(xml_path))
```

Comment:
should we even demonstrate this method now?

Reply:
removed